.github/workflows/e2e-tests.yaml

```diff
@@ -0,0 +1,93 @@
+name: E2E - Run tests
+
+on:
+  workflow_dispatch:
+  # push:
+  #   branches:
+  #     - main
+  #   tags:
+  #     - "*"
+  # pull_request:
+
+jobs:
+  supported-releases:
+    name: Build matrix for supported ADSys and Ubuntu releases
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-supported-releases.outputs.matrix }}
+    steps:
+      - name: Install needed binaries
+        run: |
+          sudo DEBIAN_FRONTEND=noninteractive apt update
```

Comment:
no need for this env variable on apt update (yeah, maybe it’s on another workflow I wrote, but shhhhh)